src/NuGet.config

```diff
@@ -1,16 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <!-- NOTE: Leave this file here and keep it in sync with list in dir.props. -->
-  <!-- The command-line doesn't need it, but the IDE does.                    -->
-  <packageSources>
-    <clear/>
-    <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
-    <add key="myget.org dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-  </packageSources>
-  <config>
-    <add key="repositoryPath" value="..\packages" />
```

Comment:
I wonder if this is still needed...  Where else does this get set?

Reply:
Not sure how to properly test this.

When I was making this change, I opened up the System.Collections solution in VS, set a bogus source in dir.props to force an early exit (and to see what command's running), and tried running/reloading while modifying this file to see what changed. I saw that during an attempted restore during build, it picks up properties from dir.props and passes them to `dotnet restore`. `repositoryPath` is effectively the same as passing `--packages`, which BuildTools uses `PackagesDir` for.

The VS NuGet tooling shows sources from this file, so that's blank now. I don't know if it's useful to anyone to have sources listed there, or if that affects a different type of restore.

To be safe, could just leave this file alone. Removing NuGet.Configs can certainly be done later.